src/snekmate/governance/TimelockController.vy

```diff
@@ -443,7 +443,7 @@ def schedule_batch(targets: DynArray[address, _DYNARRAY_BOUND], amounts: DynArra
            Must be greater than or equal to the minimum delay.
     """
     access_control._check_role(PROPOSER_ROLE, msg.sender)
-    assert len(targets) == len(amounts) and len(targets) == len(payloads), "TimelockController: length mismatch"
+    assert ((len(targets) == len(amounts)) and (len(targets) == len(payloads))), "TimelockController: length mismatch"
```

Comment:
seems unnecessary. if the precedence is wrong somehow, the code won't typecheck (unlike in python).

Reply:
As I state in the `CHANGELOG` of this PR "it doesn't follow Python's conventions, but I like it more like this". It might seem unnecessary to you, but I like it more since it gives me a clear visual distinction.